src/components/FormValidator.js

```diff
@@ -84,5 +84,3 @@ class FormValidator {
     this._setEventListeners(this._formElement, this._config);
```

Comment:
`this._formElement, this._config` are not needed here


`    this._setEventListeners();
`